vizro-core/src/vizro/models/_components/container.py

```diff
@@ -56,6 +56,9 @@ def build(self):
             components_container[f"{self.layout.id}_{component_idx}"].children = component.build()
         return html.Div(
             id=self.id,
-            children=[html.H3(children=self.title, className="container__title"), components_container],
+            children=[
+                html.H3(children=self.title, className="container__title", id=f"{self.id}-title"),
```

Comment:
I saw that too. We have been switching around often. This uses the [BEM convention](https://css-tricks.com/bem-101/), which at some point we wanted to implement because it's the recommended CSS convention. It was also the one Pruthvi mentioned we should use.

However, because neither dash, bootstrap nor mantine use BEM, at some point we said that we should just stick with dashes. So we have this mix now 😄 I can tidy this up in a separate PR, but just know we would consciously deviate from the recommended best practices to stay consistent with the other dash-libraries.

Reply:
Hmmm yes, I thought this was a left over bit of BEM. I think we should stay consistent with the other Dash libraries and tidy this up to go but definitely not a big deal and not for this PR. In fact, might be a good candidate for a Grace Hopper ticket? I'm trying to get some very simple examples like this: https://github.com/mckinsey/vizro/issues/721.

Reply:
Yes that works!

Reply:
https://github.com/mckinsey/vizro/issues/722